modules/nf-core/xeniumranger/import-segmentation/main.nf

```diff
@@ -0,0 +1,81 @@
+process XENIUMRANGER_IMPORT_SEGMENTATION {
+    tag "$meta.id"
+    label 'process_high'
+
+    container "nf-core/xeniumranger:3.0.1"
+
+    input:
+    tuple val(meta), path(xenium_bundle)
+    val(expansion_distance)
+    path(coordinate_transform)
+    path(nuclei)
+    path(cells)
+    path(transcript_assignment)
+    path(viz_polygons)
+
+    output:
+    tuple val(meta), path("**/outs/**"), emit: outs
```

Comment:
What does the tool output? different file formats should also have different output channels

Reply:
Hi Matthias, the output for xeniumranger modules is reupdated in the input files just like other 10x tools, multiple files are updated for every run and hence we do not have a dedicated channel for the outputs